### ISSUE TITLE
ENH edit legend in xsreader.xsections.plot

### ISSUE THREAD

**Issue by nicoloabrate** [CORE-GATECH-GROUP/serpent-tools#401]
**Little enhancement request**
Dear serpentTools Team,
I'd like to suggest you a very little but cool enhancement regarding the xsreader.xsections[].plot method. It would be nice to add an input parameter _label_ to control the legend label referred to that particular data. In my understanding the code does it automatically based on _MTdescrip_ attribute.

**Desired results**
I'd like to compare the xs coming from different libraries (or different temperature for the same library, and so on...) just plotting each xs with its own label
`fig, ax = plt.subplots()`
`xsJ.xsections['i94239_06c'].plot(mts=[102], loglog=True, ax=ax, label='JEFF-3.3')`
`xsE.xsections['i94239_06c'].plot(mts=[102], loglog=True, ax=ax, label='ENDF-B/VIII.0')`

Thank you for your attention!

**Comment by drewejohnson** [comment]
Hey @nicoloabrate, thanks for the suggestions! This should be a pretty easy fix, we just have to be careful about handling the label if multiple mts are provided. I'll take a crack at this shortly.

In the mean time, the following should work
```python
ax = xsJ.xsections['i94239_06c'].plot(mts=[102], loglog=True, label='JEFF-3.3')
xsE.xsections['i94239_06c'].plot(mts=[102], loglog=True, ax=ax, label='ENDF-B/VIII.0')
ax.legend(["JEFF-3.3", "ENDF-B/VIII.0"])
```

**Comment by nicoloabrate** [comment]
Thank you @drewejohnson, your suggestion is great!

**Comment by drewejohnson** [comment]
Closed with #402, merged into develop with #408
